Makefile

```diff
@@ -1,5 +1,7 @@
 CXXFLAGS ?= -I/usr/local/include
 LDFLAGS ?= -L/usr/local/lib
+# To statically link, replace the existing LDLIBS line with this:
```

Comment:
Actually, you should be able to do the substitution on the command line:

```
LDLIBS=/full/path/to/libprotobuf.a make
```

Reply:
Updated.